apps/openmw/mwmechanics/actors.cpp

```diff
@@ -1280,6 +1280,8 @@ namespace MWMechanics
                                 {
                                     if (it->first == iter->first)
                                         continue;
+                                    if (iter->first == getPlayer() && MWBase::Environment::get().getWorld()->isFirstPerson())
```

Comment:
If I understand the code correctly, this condition would be better moved to the "if" above (i.e. after line 1277), because if this is false, then the whole for loop doesn't have to be run.